NitroxPatcher/Patches/Dynamic/ExosuitGrapplingArm_OnUseUp_Patch.cs

```diff
@@ -10,6 +10,6 @@ public sealed partial class ExosuitGrapplingArm_OnUseUp_Patch : NitroxPatch, IDy
 
     public static void Prefix(ExosuitGrapplingArm __instance)
```

Comment:
Is there a reason some arm patches are prefix and other postfix? Is there a reason to not make them all postfix? @team do we have a guideline what patch to use if it's not important?

Reply:
They all use prefix unless postfix is required or if the patch reads fields that were modified inside the original method.